relay-metrics/src/aggregator.rs

```diff
@@ -1163,7 +1114,7 @@ impl AggregatorService {
     /// Split the provided buckets into batches and process each batch with the given function.
     ///
     /// For each batch, log a histogram metric.
-    fn process_batches<F>(&self, buckets: impl IntoIterator<Item = Bucket>, mut process: F)
+    pub fn process_batches<F>(&self, buckets: impl IntoIterator<Item = Bucket>, mut process: F)
```

Comment:
This is a helper function of `try_flush`, so it should be a private method of `AggregatorService`.